draft-ietf-quic-transport.md

```diff
@@ -1257,6 +1268,27 @@ frames), the receiving peer MAY send an ACK frame after a reasonable number
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
+## Special Considerations for Path MTU Discovery 
+
+Traditional ICMP-based path MTU discovery in IPv4 ({{!RFC1191}} is potentially
+vulnerable to off-path attacks that succesfully guess the IP/port 4-tuple and
+reduce the MTU to a bandwidth-inefficient value. TCP connections mitigate this
+risk by using the (at minimum) 8 bytes of transport header echoed in the ICMP
+message to validate the TCP sequence number as valid for the current
+connection. However, as QUIC operates over UDP, in IPv4 the echoed information
+may consist only of the IP and UDP headers, which usually has insufficient
```

Comment:
Routers have the option of echoing more than 28 bytes of IP packet, and I've seen pcaps where this is the case.